Initium-Odp/src/com/universeprojects/miniup/server/GameUtils.java

```diff
@@ -1087,6 +1087,20 @@ else if (iconUrl!=null && iconUrl.startsWith("http")==false)
 			requirements=true;
 			sb.append("			<div " + (isComparisonItem ? "" : "name='dexterityPenalty' ") + "class='item-popup-field' title='This is the percentage that the wearer`s dexterity will be reduced when making dexterity based rolls. Dexterity penalties stack.'>Dexterity penalty: <div class='main-item-subnote'>"+field+"%</div></div>\r\n");
 		}
+		
+		field = item.getProperty("strengthModifier");
+		if (field!=null && field.toString().trim().equals("")==false)
+		{
+			requirements=true;
+			sb.append("			<div " + (isComparisonItem ? "" : "name='strengthModifier' ") + "class='item-popup-field' title='This is the percentage that the wearer`s strength will be modified when making strength based rolls. Strength modifiers stack.'>Strength modifier: <div class='main-item-subnote'>"+field+"%</div></div>\r\n");
```

Comment:
Better to use `&apos;` instead of ` character in the title attribute. Looks cleaner.